lib/tasks/email_domain_actions.rake

```diff
@@ -0,0 +1,144 @@
+# -*- encoding : utf-8 -*-
+namespace :email_domain_actions do
+
+  desc "Lists top 20 most used email domains"
+  task :top20 => :environment do
+    # the limit of 20 is fairly arbitrary but I imagine this being more
+    # annoying than useful otherwise
```

Comment:
I don't think there should be a limit. I If I want to limit it I can just run `rake email_domain_actions:count_by_domain | head -n 20`